druid-powered.md

```diff
@@ -101,6 +101,14 @@ We are co-authors of [Turnilo](https://allegro.tech/2018/10/turnilo-lets-change-
 Our analytics platform [Storyteller](https://datumo.io/#product) uses power of Apache Druid and Turnilo to help our clients analyze large datasets in interactive manner. 
 Our biggest Druid deployment is responsible for real-time monitoring and data processing of a large video streaming platform (60 000 Events Per Second).
 
+## Deep.BI
+
+At [Deep.BI](www.deep.bi) we track user habits, engagement, product and content performance — processing up to terabytes or billions of events of data daily. Our goal is to provide real-time insights based on custom metrics from a variety of self-created dimensions. To accomplish this, our system collects every user interaction. We use Apache Flink for event enrichment, custom transformations, aggregations and serving machine learning models. The processed data is then indexed by Apache Druid for real-time analytics and our custom UI built on top of Druid and Apache Cassandra for delivery of the scores.
```

Comment:
Hey @fjy, I've forgotten to put `https://` before `www.deep.bi` in the first line which is causing the link to redirect to `https://druid.apache.org/www.deep.bi`. Can this be ammended directly or will I need to submit a new branch/commit?

The new URL between parentheses would be https://www.deep.bi/ in this case if it can be ammended directly.

Thank you again!